tests/distillation/patterns/acme_universal_error_test.html

```diff
@@ -0,0 +1,9 @@
+<html gg-domain="acme_universal_error_test">
+  <head>
+    <title>ACME Universal Error Test</title>
+  </head>
+
+  <body>
+    <p gg-match="p:has-text('Click \'Sign in\' to continue')">Click 'Sign in' to continue</p>
```

Comment:
There should be another `button` in addition to this `p`. Otherwise, there user has no button to click (e.g. to repeat the sign-in process).